src/app/zap-templates/zcl/data-model/chip/operational-state-dishwasher-cluster.xml

```diff
@@ -0,0 +1,107 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+
+  <enum name="OperationalStateEnum" type="ENUM8">
+    <cluster code="0x005A"/>
+    <item name="Stopped" value="0x00"/>
+    <item name="Running" value="0x01"/>
+    <item name="Paused"  value="0x02"/>
+    <item name="Error"   value="0x03"/>
+  </enum>
+
+  <enum name="ErrorStateEnum" type="ENUM8">
+    <cluster code="0x005A"/>
+    <item name="NoError"                    value="0x00"/>
```

Comment:
Similar comment to my previous one. The values from the generic Operational State cluster (0x01 to 0x03) should not be redefined here. This enum should only include those specific to this cluster derivation (0x40 to 0x45). 

Please see what was done for the Mode Select cluster and its derivations, in this merged PR:

https://github.com/project-chip/connectedhomeip/pull/26508/files#

src/app/zap-templates/zcl/data-model/chip/dishwasher-mode-select-cluster.xml

Reply:
Sorry, I'm not sure why this was marked as resolved. Please clarify.